A one-sample t-test is used to determine whether the mean of a single sample is significantly different from a known or hypothesized population mean. We want to test if the mean net worth of billionaires in one of the industries (say, the Technology Industry) is significantly different from a hypothesized population mean.

Let's hypothesize that the average net worth of billionaires across all industries in the USA is $80 billion.
.

In [1]:
import numpy as np
from scipy import stats

In [2]:
def one_sample_t_test(data, population_mean):
    """
    Perform a one-sample t-test.

    Parameters:
    data (list or np.array): Sample data.
    population_mean (float): Hypothesized population mean.

    Returns:
    tuple: t-statistic and p-value.
    """
    # Convert data to a numpy array if it's not already
    data = np.array(data)
   
    # Perform the one-sample t-test
    t_stat, p_value = stats.ttest_1samp(data, population_mean)
   
    return t_stat, p_value

In [3]:
# Example usage with the Technology Industry data
tech_net_worth = [120.5, 75.3, 68.4, 95.6, 110.2, 89.0, 56.3, 73.8, 97.1, 82.4]
hypothesized_mean = 80

t_stat, p_value = one_sample_t_test(tech_net_worth, hypothesized_mean)
print(f"T-statistic: {t_stat:.3f}, P-value: {p_value:.3f}")

T-statistic: 1.106, P-value: 0.297


The t-statistic of 1.106 indicates how many standard deviations the sample mean is away from the hypothesized population mean of $80 billion.

Since the p-value is much higher than the common significance level of 0.05, we fail to reject the null hypothesis. This means that there is not enough evidence to conclude that the mean net worth of billionaires in the Technology Industry is significantly different from $80 billion. ​